code/game/mob/living/simple_animal/friendly/farm_animals.dm

```diff
@@ -234,13 +234,13 @@
 					new/obj/item/weapon/plough(user.loc)
 					plough_material = null
 				plough_material = null
-		else if (choice1 == "Raise Plough")
+		else if (choice1 == "Raise plough")
```

Comment:
These cases are case-sensitive and I do not see that you've made any changes to the plough_status or data that gets passed to choice1.  This would break functionality if not consistent.

Reply:
yeah some things here are broken which is what i was actually reviewing earlier today i didn't get to this though